### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by asimonsendace**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Endace/ReleaseNotes/1_2_0.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Endace Search Archive Download PCAP v3
+
+- %%UPDATE_RN%%
```
Comment (Shellyber) [inline Packs/Endace/ReleaseNotes/1_2_0.md:6]: Please update the release notes according to the changes/additions.

File: Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download_PCAP_v3.yml [hunk 2]
```diff
@@ -0,0 +1,944 @@
...
+- contextPath: Endace.Download.PCAP.FileName
+  description: Name of the File to download from EndaceProbe
+  type: string
+- contextPath: Endace.Download.PCAP.FileSize
+  description: File size in MegaBytes
+  type: string
+- contextPath: Endace.Download.PCAP.FileUser
+  description: Username of the person who has permission to download this PCAP from EndaceProbe.
+  type: string
+fromversion: 6.8.0
```
Comment (Shellyber) [inline Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download_PCAP_v3.yml:942]:
```suggestion
fromversion: 6.10.0
```

**Review by Shellyber — CHANGES_REQUESTED** [review]
@asimonsendace thank you for your contribution. 
Please sign the CLA.
Also, make sure to update your branch with our current master.
You can do that but running the script [git_pull_master_into_fork.sh](https://github.com/demisto/content/blob/master/contribution/utils/git_pull_master_into_fork.sh)

File: Packs/Endace/ReleaseNotes/1_2_0.md (hunk 1 above)
Comment (asimonsendace) [reply Packs/Endace/ReleaseNotes/1_2_0.md]: Updated

File: Packs/Endace/ReleaseNotes/1_2_0.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Endace Search Archive Download PCAP v3
+
+- Doing a cleanup of code
```
Comment (Shellyber) [inline Packs/Endace/ReleaseNotes/1_2_0.md:6]:
```suggestion
- Documentation and metadata improvements.
```

File: Packs/Whois/Integrations/Whois/test_data/microsocks
Comment (Shellyber) [inline Packs/Whois/Integrations/Whois/test_data/microsocks:1]: Please revert this change

File: Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download_PCAP_v3.yml [hunk 4]
```diff
@@ -0,0 +1,944 @@
+id: Endace Search Archive Download PCAP v3
```
Comment (Shellyber) [inline Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download_PCAP_v3.yml:1]: What's the difference between v2 and v3? 
Should v2 be deprecated? 
Also, please add the right release notes to this playbook.

**Review by Shellyber — CHANGES_REQUESTED** [review]
Please don't forget to sign the CLA and solve the conflict
And run demisto-sdk format on the playbooks, they seem a bit broken.

**Comment by Shellyber** [comment]
Hi @asimonsendace , we haven’t heard from you in a while.
Do you need any help with the pull request?

Please feel free to reach out to me here or on Slack.
Thanks again for contributing to our repo, hope to hear from you soon.

File: Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download_PCAP_v3.yml (hunk 4 above)
Comment (asimonsendace) [reply Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download_PCAP_v3.yml:1]: That is correct v2 should be deprecated. I think the only thing that got changed was that there was a "sleep" command added within the endace-get-search-status

Comment (Shellyber) [reply Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download_PCAP_v3.yml:1]: So why add a new playbook? if it is the same? 
Why not add the "sleep" command in the v2? if that's the only change?

**Comment by Shellyber** [comment]
@asimonsendace Please resolve also the conflicts you have and sign the CLA. 
Also, Please run the `demisto-sdk format` command on the playbooks.

File: Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download_PCAP.yml [hunk 5]
```diff
@@ -1,783 +1,783 @@
...
-outputs: []
-tests:
-- No test - Deprecated playbook
-fromversion: 5.0.0
+id: Endace Search Archive Download PCAP
+version: -1
+hidden: true
+name: Endace Search Archive Download PCAP
+description: Deprecated. This playbook has been deprecated. Use Endace Search Archive Download\
+  \ PCAP v2 instead. This playbook uses Endace APIs to search, archive and download\
```
Comment (ssokolovich) [inline Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download_PCAP.yml:6]:
```suggestion
  \ PCAP v3 instead. This playbook uses Endace APIs to search, archive and download\
```
```suggestion
  \ PCAP v2 instead. This playbook uses Endace APIs to search, archive and download\
```

File: Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download_PCAP_README.md [hunk 6]
```diff
@@ -1,51 +1,51 @@
...
-| archive_filename | Name of the archive file. For example, archive\_filename could be an event ID. To keep archive filename unique, value of epoch seconds at the time of execution of the command is appended to this filename argument. For example \- if the event id is 123456789, then archive\_filename is 123456789\-&lt;epochtime&gt;. |  | Optional |
-| delete_archive | Delete archived file 0: Don't delete archived file. 1: Delete archived file. By default archived files on EndaceProbe won't be deleted |  | Optional |
-
-## Playbook Outputs
----
-There are no outputs for this playbook.
-
-## Playbook Image
----
+DEPRECATED - This playbook has been deprecated. Use Endace Search Archive Download PCAP v2 instead. This playbook uses Endace APIs to search, archive and download PCAP file from either a single EndaceProbe or many via the InvestigationManager.      The workflow accepts inputs like “the date and time of the incident or a timeframe”, “source or destination IP address of the incident”,  “source or destination IP port of the incident”,  “protocol of the incident” and name of archive file.
```
Comment (ssokolovich) [inline Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download_PCAP_README.md:1]:
```suggestion
DEPRECATED - This playbook has been deprecated. Use Endace Search Archive Download PCAP v3 instead. This playbook uses Endace APIs to search, archive and download PCAP file from either a single EndaceProbe or many via the InvestigationManager.      The workflow accepts inputs like “the date and time of the incident or a timeframe”, “source or destination IP address of the incident”,  “source or destination IP port of the incident”,  “protocol of the incident” and name of archive file. 
```
```suggestion
DEPRECATED - This playbook has been deprecated. Use Endace Search Archive Download PCAP v2 instead. This playbook uses Endace APIs to search, archive and download PCAP file from either a single EndaceProbe or many via the InvestigationManager.      The workflow accepts inputs like “the date and time of the incident or a timeframe”, “source or destination IP address of the incident”,  “source or destination IP port of the incident”,  “protocol of the incident” and name of archive file. 
```

**Review by ssokolovich — COMMENTED** [review]
Hey @asimonsendace, waiting for you to perform `demisto-sdk format` so I can check the rest of the files.
Please note that per @Shellyber comment, if there isn't a significant change in the playbook logic between v2 and v3 -> we should modify the v2 instead of creating a new one.

Cheers!

**Comment by Shellyber** [comment]
Hi @asimonsendace any update here?

**Comment by Shellyber** [comment]
Hi @asimonsendace,
Thank you for your contribution. 
I have not received a response from you in the last two weeks, so I will close your PR now.
Please feel free to re-open it when you are available to continue. 
Thanks again.
